tests/components/honeywell/test_climate.py

```diff
@@ -54,7 +54,7 @@ async def test_no_thermostats(
     """Test the setup of the climate entities when there are no appliances available."""
     device._data = {}
     await init_integration(hass, config_entry)
-    assert len(hass.states.async_all()) == 0
+    assert len(hass.states.async_all()) == 2  # 0 climate entities; 2 sensor entities
```

Comment:
This will be 3 after the PR - If it's merged before this you will need to rebase.
Until then, this is correct.

Reply:
Please rebase as https://github.com/home-assistant/core/pull/98392 is merged